### PR TITLE
Allow importing files to project dialog box

### PR DESCRIPTION
**Pull request by turtletooth**
#1017 

When loading an external file, this pops up dialog allowing you to choose whether to load the file from its current location or import the file to the current project folder.  There is a checkbox to remember the choice.  This is also in Preferences->Loading.

### CODE REVIEW

File: toonz/sources/toonzqt/dvdialog.cpp [hunk 1]
```diff
@@ -757,6 +758,29 @@ void Dialog::addButtonBarWidget(QWidget *first, QWidget *second, QWidget *third,
...
+                                                   bool hasButton,
+                                                   bool hasFixedSize,
+                                                   const QString &name)
+    : Dialog(parent, hasButton, hasFixedSize, name) {}
+
+//=============================================================================
+
+void MessageAndCheckboxDialog::onButtonPressed(int id) {
+  if (m_checked > 0) {
+    done(id + 5);
```
Comment (shun-iwasawa) [inline toonz/sources/toonzqt/dvdialog.cpp]: You are returning the state of checkbox by adding 5 to the return value.
It's not a problem for the moment, but if the dialog is created with more than 5 buttons, return value will be overlapped. 
How about replacing this magic number with a larger & named constant (say, `const static int  MessageAndCheckboxDialog::CHECKED = 1000` ) in order to surely prevent the overlap? Or, how about adding a separate function which will return `m_checked` value?

**Comment by turtletooth** [comment]
I made a new function to communicate the checked state.
Thanks @shun-iwasawa for the review.

**Comment by shun-iwasawa** [comment]
Jenkins

**Comment by shun-iwasawa** [comment]
It works fine. Thank you @turtletooth ! LGTM

**Pull request merged**